bin/ci-builder

```diff
@@ -17,7 +17,15 @@ cd "$(dirname "$0")/.."
 
 if [[ $# -lt 2 ]]
 then
-    echo "usage: $0 <build|push|run> <stable|nightly> [<args>...]"
+    echo -n "usage: $0 <build|push|run> <stable|nightly> [<args>...]
+
+Subcommands:
+
+    run        run a command inside the ci-builder in the current directory.
```

Comment:
It actually always runs relative to the root of the repository. (Maybe that's what you meant? But seemed ambiguous!)

Reply:
I didn't actually realize that, I have always run it from the root.